content/source/docs/enterprise/private/updating-license.html.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: "enterprise2"
+page_title: "Updating License Key"
+sidebar_current: "docs-enterprise2-private-installer-rhel"
+---
+
+# Updating License Key
+
+This guide is for updating the License file for an installation of Private Terraform Enterprise.
```

Comment:
"license" and "license key" should be lowercase. 

This intro sentence should include a brief word about when an admin would want to/be able to update a license file.

Reply:
Fixed! 👍